packages/samples/specs/authentication/main.tsp

```diff
@@ -2,6 +2,7 @@ import "@typespec/rest";
 
 using TypeSpec.Http;
 
+#suppress "@azure-tools/typespec-autorest/unsupported-http-auth-scheme" "BearerAuth is not supported by typespec-autorest"
```

Comment:
This doesn't feel great that we have to suppress here to satisfy autorest. Maybe we can exclude that spec from running in the typespec-azure repo instead.

Reply:
Agree, I fear folks looking at this sample would be confused that bearer auth is not supported by TypeSpec somehow, or otherwise just confused about why this is here.

Reply:
Sounds good, I'll exclude this sample from that part of the tests.